.circleci/config.yml

```diff
@@ -0,0 +1,136 @@
+version: 2.1
+
+
+commands:
+  npm-install:
+    steps:
+      - run: sudo npm install npm@6 --global
+      - restore_cache:
+          key: js-deps-{{ checksum "package.json" }}
```

Comment:
If I understand correctly, the npm install can happen on different versions of Node JS and the cache can be loaded from another version? If the test-node12 step saves cache before the restore cache line happens on test-node10 then any compiled dependencies for node12 may cause node 10 to fail.